src/playlist-loader.js

```diff
@@ -9,6 +9,7 @@ import resolveUrl from './resolve-url';
 import {mergeOptions} from 'video.js';
 import Stream from './stream';
 import m3u8 from 'm3u8-parser';
+import {window} from 'global';
```

Comment:
The build shows that it works. I think that `import window from 'global/window';` makes sense too and if you have a strong preference to it I can change it. I think we use it similarly in other projects. I don't really like `import window from 'global'` as it may be confusing since global can be used to require more than just window.